packages/terra-notification-dialog/src/orion-fusion-theme/NotificationDialog.module.scss

```diff
@@ -0,0 +1,48 @@
+:local {
+  .orion-fusion-theme {
+    --terra-notification-dialog-background: #fff;
+    --terra-notification-dialog-border-radius: 0;
+    --terra-notification-dialog-small-horizontal-inset: 10px;
+    --terra-notification-dialog-max-width: 500px;
+    --terra-notification-dialog-floating-background-height: 3rem;
+    --terra-notification-dialog-alert-background-image: linear-gradient(to bottom, #fce5e5, #fff);
+    --terra-notification-dialog-alert-border-top: 3px solid #e50000;
+    --terra-notification-dialog-error-background-image: linear-gradient(to bottom, #f6f6f6, #fff);
+    --terra-notification-dialog-error-border-top: 3px solid #9b9fa1;
+    --terra-notification-dialog-warning-background-image: linear-gradient(to bottom, #fff2d0, #fff);
+    --terra-notification-dialog-warning-border-top: 3px solid #ffc20a;
+    --terra-notification-dialog-info-background-image: linear-gradient(to bottom, #f6f6f6, #fff);
+    --terra-notification-dialog-info-border-top: 3px solid #9b9fa1;
+    --terra-notification-dialog-custom-background-image: linear-gradient(to bottom, #f6f6f6, #fff);
+    --terra-notification-dialog-custom-border-top: 3px solid #9b9fa1;
+    --terra-notification-dialog-header-padding: 0 0.83333rem;
+    --terra-notification-dialog-header-content-border-bottom: 1px solid #9b9fa1;
+    --terra-notification-dialog-header-content-padding: 0.75rem 0 0.83333rem;
+    --terra-notification-dialog-header-container-margin-left: 0.58333rem;
+    --terra-notification-dialog-signal-word-color: #1c1f21;
+    --terra-notification-dialog-signal-word-font-size: 1rem;
+    --terra-notification-dialog-signal-word-min-height: 1rem;
+    --terra-notification-dialog-title-color: #2d3539;
+    --terra-notification-dialog-title-font-size: 1rem;
+    --terra-notification-dialog-title-font-weight: normal;
+    --terra-notification-dialog-title-margin: 0.08333rem 0 0;
+    --terra-notification-dialog-title-min-height: 1rem;
```

Comment:
Needs `terra-notification-dialog-title-margin-top` to satisfy #1175 .

Reply:
I double-checked my changes, `terra-notification-dialog-title-margin-top` styles have removed: https://github.com/cerner/terra-framework/pull/1105/files#diff-199a5a3214c6de6b926614643935d3a5L102